conf/messages.en

```diff
@@ -2074,3 +2074,13 @@ existingSiteClaimingAmount.cya.agent = How much does your client want to claim f
 
 newSpecialTaxSites.cya = Total new enhanced structures and buildings allowance claim:
 newSpecialTaxSites.change.hidden =
+
+########## National Insurance Contributions - Class 2 ##########
+
+nics.class2.title = Voluntary Class 2 National Insurance contributions
+nics.class2.p1 = As your adjustable taxable profit is less than £{0} you don’t need to make Class 2 National Insurance contributions.
+nics.class2.p2 = However, you may still want to pay these to get a higher State Pension entitlement.
```

Comment:
Should these have agent specific text i.e. 'However, your client may still ...' ?

Reply:
Bobby is updating the prototype, I will update it as soon as he gets back to me.